app/mailers/user_mailer.rb

```diff
@@ -92,7 +92,7 @@ def subject_for(key)
       default: [:subject, key.to_s.humanize], app_name: app_name)
   end
 
-  def preview?
-    GovukAdminTemplate.environment_label == "Preview"
+  def production?
+    Rails.application.config.instance_name.blank?
```

Comment:
We made the right pragmatic decision here, but relying on `instance_name` being blank feels like it might bite us in the future.